application/blocks/bestitamazonpay4oxid_order_confirm_button.tpl

```diff
@@ -51,40 +51,46 @@
                 var $form = $(this);
                 var $classInput = $('input[name="cl"]', $form);
                 var $functionInput = $('input[name="fnc"]', $form);
+                var $agbCheckbox = $('#checkAgbTop');
```

Comment:
I think the check should be on input name instead of id, because there can be an additional #checkAgbBottom and/or different ids per template.
$('input[name="ord_agb"]', $form) should handle all of the possibilities.

Reply:
Changed it